thingsboard_gateway/storage/sqlite/database.py

```
@@ -67,30 +67,34 @@ def __init__(self, config, processing_queue: Queue, logger, stopped: Event):
         self.__next_batch = []
         self.__initialized = True
 
-        self.start()
-
     def init_table(self):
         try:
-            # Check if the old schema exists
             try:
+
                 result = self.db.execute_read(
                     "SELECT sql FROM sqlite_master WHERE type='table' AND name='messages';"
                 ).fetchone()
+
+            except Exception as e:
+                result = None
```

It may fail silently here.